types/src/jsonrpc/wrapped/batches.rs

```diff
@@ -180,32 +178,18 @@ impl<T> BatchesState<T> {
 	/// Injects a newly-received batch into the list. You must then call
```

Comment:
```suggestion
	/// Injects a newly received batch into the list. You must then call
```

Reply:
TIL: you don't hyphenate when an adverb ends with "-ly"